editor/src/components/navigator/left-pane/index.tsx

```diff
@@ -93,7 +93,7 @@ export const LeftPaneComponent = React.memo<LeftPaneComponentProps>((props) => {
           }}
         >
           {when(
-            isMyProject,
+            isProjectOwner,
```

Comment:
this later needs to be changed to a semantic check (`canEdit`/`canViewPanes` or something along that lines), but I wanted to keep this PR focused